job.js

```diff
@@ -12,6 +12,22 @@ const BigQueryFactory = require('./factory');
 const COST_THRESHOLD_IN_GB = 100;
 const COST_PER_TB = 5;
 
+/**
+ * Enum string values.
+ * @enum {string}
+ */
+const WriteDisposition = {
+    WRITE_APPEND: 'WRITE_APPEND',
+    WRITE_EMPTY: 'WRITE_EMPTY',
+    WRITE_TRUNCATE: 'WRITE_TRUNCATE'
+};
+
+/**
+ * @typedef {Object} TempTableConfig
+ * @property {String} dataset
+ * @property {String} table
```

Comment:
Following `Dataset` and `Table` convention, it should be `datasetName` and `tableName`